bin/http-server

```diff
@@ -38,6 +38,7 @@ if (argv.h || argv.help) {
     '  -K --key     Path to ssl key file (default: key.pem).',
     '',
     '  -r --robots  Respond to /robots.txt [User-agent: *\\nDisallow: /]',
+    '  --hide-dotfiles  Don\'t show dotfiles',
```

Comment:
Fixed - I didn't know optimist had that feature. Does http-server just follow optimist's approach, or are the conventions outlined somewhere else?